src/CSharp/MetadataWebApi/MetadataWebApi/MetadataApiFactory.cs

```diff
@@ -27,15 +33,15 @@ public class MetadataApiFactory : IMetadataApiFactory
         public virtual IMetadataApi CreateMetadataApi()
         {
             // Get the token to use to connect to the QAS Electronic Updates Metadata REST API
-            string token = GetConfigSetting("Token");
+            string token = GetConfigSetting("token");
 
             if (string.IsNullOrEmpty(token))
             {
-                throw new ConfigurationErrorsException("The Electronic Updates authentication token has not been configured.");
+                throw new NullReferenceException("The Electronic Updates authentication token has not been configured.");
```

Comment:
@Lexcess Don't think we want to be throwing NullReferenceExceptions manually do we?